src/index.ts

```diff
@@ -168,8 +168,7 @@ export class Finch extends Core.APIClient {
   sandbox: API.Sandbox = new API.Sandbox(this);
 
   /**
-   * Returns an access token for the Finch API given an authorization code. An
-   * authorization code can be obtained by visiting the url returned by getAuthURL().
+   * DEPRECATED: use client.accessTokens.create instead.
```

Comment:
does this need a deprecated tag too?

Reply:
Ah, good catch. Yes that should be marked as deprecated

Reply:
@tessfinch That's fixed